### PR TITLE
AC Automated Tests - LRAC-13506 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-13506
https://issues.liferay.com/browse/LRAC-13507
https://issues.liferay.com/browse/LRAC-13508
https://issues.liferay.com/browse/LRAC-13509
https://issues.liferay.com/browse/LRAC-13489

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by manuferreira — APPROVED** [review]
LGTM. Obrigada, my friend! :tada:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase [hunk 1]
```diff
@@ -617,4 +617,102 @@ definition {
...
+		}
+
+		task ("Assert that there's only one result in the response") {
+			var searchedValue = ACApi.searchApiResponse(
+				apiDataResponse = ${apiIndividualOrAssetResponse},
+				jsonQuery = "total");
+
+			TestUtils.assertEquals(
+				actual = ${searchedValue},
+				expected = "");
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase:786]: Thanks for working on this Vini! I had a question on why this is expecting an empty response? Shouldn't it be 1?

**Comment by ViniLopes20** [comment]
Covered by: https://github.com/liferay-ac/liferay-portal/pull/405

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase [hunk 2]
```diff
@@ -451,4 +451,108 @@ definition {
...
```
[the remark this answers is not in the document]
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase:786]: Hey @joshitagaki, the expected would be really empty, the answer of the `ACApi.createIndividualOrAssetDataReport` macro that stores the result in the variable `apiIndividualOrAssetResponse` was executed using the API that used the ID of a specific document and media and in this result the `total` does not appear in the answer

To verify that the result is appearing correctly, we are verifying that the `total` only appears when we use the endpoint that will return all the results for that `type` that is being used and when we use it for a specific document of some `type`, then the `total` should not appear. What do you think?

[the remark this answers is not in the document]
Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase:786]: Oh ok got it. That makes sense. Thanks!

**Pull request closed without merging**